Sources/Risk/Services/RiskError.swift

```diff
@@ -1,35 +1,52 @@
 //
 //  RiskError.swift
-//  
+//
 //
 //  Created by Precious Ossai on 13/10/2023.
 //
 
 import Foundation
 
 public enum RiskError: LocalizedError, Equatable {
-  case integrationDisabled
-  case couldNotPublishRiskData
-  case couldNotRetrieveConfiguration
-  case couldNotPersisRiskData
-  case fingerprintServiceIsNotConfigured
-
-  public var errorDescription: String? {
-    switch self {
-    case .integrationDisabled:
-      return "Integration disabled"
-
-    case .couldNotPublishRiskData:
-      return "Error publishing risk data"
-
-    case .couldNotRetrieveConfiguration:
-      return "Error retrieving configuration"
-
-    case .couldNotPersisRiskData:
-      return "Error persisting risk data"
+    case configuration(Configuration)
+    case publish(Publish)
```

Comment:
we can propagate the error descriptions here:

```public var errorDescription: String? {
      switch self {
      case .configuration(let error):
        return error.localizedDescription

      case .publish(let error):
        return error.localizedDescription
      }
    }

Reply:
Would we need to change the implementation of it as well ie. for each use case of RiskError?

Reply:
I don't believe so. This is just to give the clients an easier way to handle the errors. Not actually handling but geting the descriptive message out thereof. 

If they want to, they'll do:

`catch { print(error.localizedDescription) }`

rather than switching over all the possible cases.

Reply:
That currently works without this new change, though.

Reply:
okay then we don't need it, cool